JPS_BASE_LIB/src/main/java/uk/ac/cam/cares/jps/base/converter/README.md

```diff
@@ -214,3 +214,21 @@ jpsBaseLibGW.importPackages(jpsBaseLib_view, "uk.ac.cam.cares.jps.base.converter
 tbox_generation = jpsBaseLib_view.TBoxGeneration()
 tbox_generation.generateTBox('./<FILE_NAME>.csv')
 ```
+
+## Generating description logic in LaTeX format
+
+By default, the TBox Generator generates a `.tex` file containing the description logic of the ontology in LaTeX format in the same folder as the provided CSV file. If one wants to generate the `.tex` file for an existing OWL file, please follow below steps:
+1. Build the base lib by means of `mvn clean install -DskipTests` in the root folder of this project.
+2. Run the following command from within the `target` folder if the ontology is available online:
+    ```
+    java -cp jps-base-lib.jar uk.ac.cam.cares.jps.base.converter.DescriptionLogicGenerator --url <ontologyURL> <texFilePath>
+    ```
+    e.g. `java -cp jps-base-lib.jar uk.ac.cam.cares.jps.base.converter.DescriptionLogicGenerator --url https://raw.githubusercontent.com/cambridge-cares/TheWorldAvatar/main/JPS_Ontology/ontology/ontodoe/OntoDoE.owl ../../JPS_Ontology/ontology/ontodoe/OntoDoE.tex`
+
+    or if the ontology is available in the local file system:
+    ```
+    java -cp jps-base-lib.jar uk.ac.cam.cares.jps.base.converter.DescriptionLogicGenerator --file <ontologyFilePath> <texFilePath>
+    ```
+    e.g. `java -cp jps-base-lib.jar uk.ac.cam.cares.jps.base.converter.DescriptionLogicGenerator --file ../../JPS_Ontology/ontology/ontodoe/OntoDoE.owl ../../JPS_Ontology/ontology/ontodoe/OntoDoE.tex`
+
+**NOTE:** A list of namespaces are defined in `TheWorldAvatar/JPS_BASE_LIB/src/main/resources/ontology.namespaces.properties` to be used as short form of IRIs, one can modify this file before running the above commands to customise the LaTeX output.
```

Comment:
Please add the following notes:
1) If `ontology.namespaces.properties` is modified, the base lib jar file needs to be rebuilt in order for the changes to take effect.
2) If a namespace is not found in that file, the full IRI will appear in the LaTeX output. If the IRI contains a `#` character, as is often the case, this causes errors in LaTeX.
3) The LaTeX output may contain characters that are not supported, neither in `\usepackage[latin1]{inputenc}` nor `\usepackage[utf8]{inputenc}` encoding (which is the default in the CoMo preprint template). For example, `¬` and `≡` will need to be replaced with `\neg` and `\equiv`, respectively.

Reply:
Addressed in https://github.com/cambridge-cares/TheWorldAvatar/pull/736/commits/6ab3d7fe603b34b0b33e9373339bd43791b98ee5